components/ide-metrics-api/generate.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Could you leave a file like `README.md` to tell us to generate files and things worth remind etc?